app/controllers/concerns/attachment_manageable.rb

```diff
@@ -2,7 +2,24 @@ module AttachmentManageable
   def allow_only_one_attachment
     return unless params[:files].is_a?(Array)
 
-    flash.now[:alert] = t(".attachment_manageable.upload_limit_exceeded")
+    flash.now[:alert] = t("attachment_manageable.upload_limit_exceeded")
     render turbo_stream: turbo_stream.replace("flash", partial: "layouts/shared/flash_messages")
   end
+
+  def show_alert_if_attachment_missing
+    if !params[:pet][:images].nil?
+      no_empty_images = params[:pet][:images].reject { |image| image == "" }
```

Comment:
`no_empty_images` could just be renamed `images`, same for `files`.

Reply:
Did this in [refactor: change no_empty_images to images and files](https://github.com/rubyforgood/homeward-tails/pull/1273/commits/22d84b32d7593e16f3339229da03991124a44a7f)